constantcontact.com.email.json

```diff
@@ -8,19 +8,28 @@
   "syncRedirectDomain": "app.constantcontact.com",
   "logoUrl": "https://static.ctctcdn.com/lp/images/standard/logos/ctct_logo_full-color.svg",
   "description": "Enables you to setup DKIM email authentication for emails sent from your domain",
-  "variableDescription": "sel1/sel2 and ctctsel1/ctctsel2 are selector values; domain_prefix identifies the target CTCT domain",
+  "variableDescription": "sel1/sel2 and ctctsel1/ctctsel2 are selector values; domain_prefix identifies the target CTCT domain; dmarc is the dmarc record value",
   "records": [
     {
+      "groupId": "dkim",
       "type": "CNAME",
       "host": "%sel1%._domainkey",
       "pointsTo": "%ctctsel1%._domainkey.dkim1.%domain_prefix%send.com",
       "ttl": 3600
     },
     {
+      "groupId": "dkim",
       "type": "CNAME",
       "host": "%sel2%._domainkey",
       "pointsTo": "%ctctsel2%._domainkey.dkim2.%domain_prefix%send.com",
       "ttl": 3600
+    },
+    {
+      "groupId": "dmarc",
```

Comment:
I suggest adding `txtConflictMatchingMode` = `All` for DMARC in order to avoid multiple entries. Consider also an appropriate setting of `essential` for this record if someone would like to edit it or overwrite.

Reply:
thank you, adding `txtConflictMatchingMode`; re "essential", it's not clear to me which of the 2 values in the spec  (`Always`/`OnApply`) would allow overwrite, can you clarify? by 'someone' do you mean a manual edit or an edit via re-applying the template?

Reply:
pls let me know if to keep the current version or you want any more changes.

Reply:
current version is good, thanks!